setup.py

```diff
@@ -226,9 +226,9 @@ def run(self):
         ]
     ),
     cmdclass=dict(
-        type_check=TypeCheckCommand,
-        style_check=StyleCheckCommand,
-        build_py=BuildPyCommand,
+        # type_check=TypeCheckCommand,
```

Comment:
I guess this needs to be removed.

Reply:
Unless the code in the current PR is affected by this, can we handle this a separate PR? I can prepare something that removes these checks from the build chain and adds them to the Jenkins pipeline instead.